packages/pangea-sdk/PangeaCyber.Net/Audit/AuditClient.cs

```diff
@@ -480,6 +480,12 @@ private string GetPublicKeyData()
             }
         }
 
+        ///
```

Comment:
Docs? Or is someone else gonna write them?

Reply:
Yes. Once the API is almost finished, I'll assign a ticket to you